freqtrade/rpc/api_server/api_v1.py

```diff
@@ -259,3 +259,7 @@ def list_available_pairs(timeframe: Optional[str] = None, stake_currency: Option
         'pair_interval': pair_interval,
     }
     return result
+
+@router.get('/sysinfo', tags=['info'])
+def sysinfo(rpc: RPC = Depends(get_rpc)):
```

Comment:
using `Depends(get_rpc)` will mean it'll not work for webserver mode ... which is unnecessary in this case.